src/_pytest/mark/structures.py

```diff
@@ -205,17 +194,25 @@ def test_function():
 
     mark = attr.ib(validator=attr.validators.instance_of(Mark))
 
-    name = alias("mark.name")
-    args = alias("mark.args")
-    kwargs = alias("mark.kwargs")
+    @property
+    def name(self):
+        """alias for mark.name"""
+        return self.mark.name
+
+    @property
+    def args(self):
+        """alias for mark.args"""
+        return self.mark.args
```

Comment:
This appears to be unused also, at least not covered.

Reply:
I added a test for the aliases.